src/instructlab/model/backends/backends.py

```diff
@@ -105,6 +107,9 @@ def shutdown(self):
     def register_resources(self, resources: typing.Iterable[Closeable]) -> None:
         self.resources.extend(resources)
 
+    def get_backend_type(self):
```

Comment:
Would it be better to make this an @abc.abstractmethod, not have the attribute, and return a constant for each of the subclasses get_backend_type() method.

Reply:
good point! That would solve my linting issue too